lua/nvim-tree/core.lua

```diff
@@ -31,10 +34,12 @@ function M.reset_explorer()
   TreeExplorer = nil
 end
 
+---@return string
```

Comment:
This should be `string|nil`, but many internal functions rely on always receiving a string, so that's something I need to fix.